app/controllers/application_controller.rb

```diff
@@ -35,4 +35,10 @@ def require_active_current_user
     require_current_user
     redirect_to edit_profile_path if @current_user.present? && !@current_user.active?
   end
+
+  def require_privilege
+    unless @current_user.has_privilege?(controller_name, action_name)
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.